rust-toolchain.toml

```diff
@@ -13,5 +13,5 @@
 # limitations under the License.
 
 [toolchain]
-channel = "nightly-2024-03-05"
+channel = "stable"
```

Comment:
Thank you! This guards the result.

I'm not quite familiar with stable, so I wonder if "1.76.0" more clear than "stable" and we may  bump explicitly?

Reply:
Actually no, just *stable* is the right answer here. One stabilized language features are not removed and you don't want to lock it to an older toolchain than the current stable one.

What you are after might be the MSRV value in Cargo.toml `[package.rust-version]`. That you can use to specify an exact minimum version.

Reply:
Thanks for your information @alerque!

Reply:
Added. `rust-version`.

Thanks again for your sharing :D